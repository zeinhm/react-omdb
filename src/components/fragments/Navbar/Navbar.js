import React from 'react'
import { Link } from 'react-router-dom'
import reactLogo from '../../../assets/react-logo.png'
import './styles.scoped.css'

export default function Navbar() {
  return (
    <div className='navbar'>
      <Link to='/'><span><img src={reactLogo} alt=""/></span> React OMDb</Link>
    </div>
  )
}
