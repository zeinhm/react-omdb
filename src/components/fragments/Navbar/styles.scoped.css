.navbar {
  align-items: center;
  display: flex;
  justify-content: space-between;
  margin: auto 2rem;
  padding: 1rem;
  width: fit-content;
}

.navbar a {
  align-items: center;
  background: linear-gradient(to right top, #16BFFD 0%, #CB3066  90%);
  border-radius: 10px 0;
  box-shadow: 0 1px 5px rgba(199, 119, 185, 0.75);
  color: #fff;
  display: flex;
  padding: 1rem 2rem;
  text-decoration: none;
  transition: 0.5s;
}

.navbar a:hover {
  box-shadow: 0 4px 8px rgb(209, 107, 165);
  color: #fff;
  text-decoration: none;
  transition: 0.5s;
}

.navbar span {
  align-items: center;
  display: flex;
}

.navbar a img {
  height: 1.5rem;
}