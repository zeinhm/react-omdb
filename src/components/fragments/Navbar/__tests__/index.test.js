import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Navbar from '../Navbar';

describe('src/components/fragments/Navbar', ()=> {
  test('Navbar', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<Navbar />);
    expect(tree).toMatchSnapshot();
  });
});
