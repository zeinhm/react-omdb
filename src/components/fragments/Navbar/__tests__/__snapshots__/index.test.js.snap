// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/fragments/Navbar Navbar 1`] = `
<div
  className="navbar"
>
  <Link
    to="/"
  >
    <span>
      <img
        alt=""
        src="react-logo.png"
      />
    </span>
     React OMDb
  </Link>
</div>
`;
